pub/index.html

```diff
@@ -0,0 +1,86 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Maddie's Blog</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="apple-touch-icon" href="apple-touch-icon.png">
+        <!-- Place favicon.ico in the root directory -->
+
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/main.css">
+        <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+    </head>
+    <body>
+        <header>
+            <img style="width:100%;" src="images/banner.jpg">
+        </header>
+        <div style="width:1060px; margin: 0 auto;">
+            <aside style="float:left; width:25%; box-sizing:border-box; padding-right:60px;">
+                <div class="aside-content" style="border-bottom: solid 1px #e8e8e8; padding-bottom: 10px; margin-bottom: 25px;">
+                    <img style="margin-top:-68px; margin-bottom:20px; position:relative;" src="images/m-2.png">
+                    <h1 style="color:#666666; font-size:16px; margin-bottom:18px; font-family:'Open-sans' sans-serif; text-transform:uppercase; font-weight:bold;">Maddie's Blog  |  <br> Cool Shit</h1>
+                    <p style="font-size:17px; font-family:'PT serif' serif; color:#bbb; line-height:26px;">A small gathering of information about my life</p>
+                </div>
+
+                <form>
+                    <h2 style= "color:#bbb; font-size:14px; margin-bottom:18px; font-family:'Open-sans' sans-serif; text-transform:uppercase; font-weight:bold;">Search</h2>
+                    <input style="width: 100%; height: 25px; border-radius: 2px; border: solid 1px #e8e8e8;" type="text">
+                </form>
+            </aside>
+            <section style="float:left; width:75%; box-sizing:border-box; margin-top:62px;">
+                <article style="border-bottom: solid 2px #e8e8e8; padding-bottom:38px;">
+                    <h1 style="color:#39474e; font-size:40px; font-family:'Open-sans' sans-serif; font-weight:bolder;">My Boyfriend</h1>
+                    <blockquote style="display:block; margin-top:32px;">
+                        <p style="font-size:17px; font-family:'PT serif' serif; color:#bbb; line-height:26px; font-style:italic; border-left:solid 2px #e8e8e8; padding-left:15px;"><em>Maddie and Kolby</em></p>
+                    </blockquote>
```

Comment:
Better as an `<address>` tag, but don't tell Susanna... She's ecstatic about writing blog posts as it is.